### PR TITLE
added blocking commands tests

### PR DESCRIPTION
**Pull request by avifenesh**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: redis/tests/test_blocking_commands.rs
Comment (ikolomi) [inline redis/tests/test_blocking_commands.rs:1]: I dont think a single tests justifies having it in a new file, its better to keep them together in the test_cluster_async.rs

Comment (avifenesh) [reply redis/tests/test_blocking_commands.rs:1]: To my understanding we are planing to create state restoration at some point, so i thought it is better to create a new file ready to add massive tests when we will have the support. can move back if you think its better.

File: redis/tests/test_blocking_commands.rs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#![cfg(feature = "cluster-async")]
+mod support;
+
+#[cfg(test)]
+mod test_blocking_commands {
+    use redis::{AsyncCommands, RedisResult, Value};
+
+    use crate::support::*;
+    #[tokio::test]
+    async fn test_blocking_command_when_cluster_drops() {
```
Comment (ikolomi) [inline redis/tests/test_blocking_commands.rs:10]: why to invent a different logic and form - copy paste and modify:  https://github.com/amazon-contributing/redis-rs/blob/4571c91e2165b9ebf3bb9a3dff0991bc453d1075/redis/tests/test_cluster_async.rs#L3680

Comment (ikolomi) [reply redis/tests/test_blocking_commands.rs:10]: you can add your test below that one

**Review by ikolomi — CHANGES_REQUESTED** [review]

**Review by ikolomi — APPROVED** [review]

**Pull request merged**